modules/configloader/src/BoardConf.cxx

```diff
@@ -0,0 +1,147 @@
+/*
+ *  BoardConf.cxx
+ *  Copyright 2020 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <concepts>
+#include <fstream>
+#include <optional>
+#include <nameof.hpp>
+#include <range/v3/algorithm/transform.hpp>
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wambiguous-reversed-operator"
```

Comment:
Compiler specific preprocessor directives to silence C++2a compatibility warnings. rapidjson is not C++2a ready; see the rapidjson patch in `SetupUrho3D.cmake` for more information.